NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -1,37 +1,848 @@
 ﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
 using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media.Animation;
+using Microsoft.WindowsAPICodePack.Dialogs;
 using NitroxLauncher.Models;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Server;
+using NitroxServer.Serialization;
+using NitroxServer.Serialization.World;
+using Microsoft.VisualBasic.FileIO;
 
 namespace NitroxLauncher.Pages
 {
     public partial class ServerPage : PageBase
     {
-        public string StartButtonSubtitle => $"NITROX {LauncherLogic.ReleasePhase} {LauncherLogic.Version}";
-        private bool IsServerExternal => LauncherLogic.Config.IsExternalServer;
+        public bool IsServerExternal
+        {
+            get => LauncherLogic.Config.IsExternalServer;
+            set => LauncherLogic.Config.IsExternalServer = value;
+        }
+
+        public ServerConfig Config;
+
+        private bool IsNewWorld { get; set; }
+        private bool IsInSettings { get; set; }
+
+        private string SelectedWorldDirectory { get; set; }
+        private string WorldCurrentlyUsed { get; set; }
```

Comment:
It's not clear that this property is a directory. Maybe `WorldDirCurrentlyUsed`?